### PR TITLE
add all files

### PR DESCRIPTION
**Pull request by tolyanchenko**
add all files

### CODE REVIEW

File: nothing.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+313213
```
Comment (usatovaDasha) [inline nothing.txt:1]: ???

File: test.css [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+.done-text {
+  font-size: 20px;
+  color: rgba(0, 0, 0, 0.479);
+  text-decoration: line-through;
+}
+
+.picParamsDelete {
+  width: 25px;
+  height: auto;
+}
```
Comment (usatovaDasha) [inline test.css:87]: add empty line

File: test.css [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+#inputEdit {
+  margin: 5px;
+  min-height: 30px;
+  padding: 5px;
+}
+
+#newButton {
+  margin-right: 5px;
+  min-height: 30px;
+}
```
Comment (usatovaDasha) [inline test.css:102]: add empty line

File: test.css [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+.checkboxDivClass {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 25px;
+  height: auto;
+  margin-right: 10px;
+}
```
Comment (usatovaDasha) [inline test.css:135]: add empty lines between blocks

File: test.css [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+
+.textDivClass {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  width: 500px;
+  height: auto;
+  /* overflow: hidden; */
```
Comment (usatovaDasha) [inline test.css:154]: remove

File: test.css [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
+  align-items: normal;
+  margin-top: 10px;
+  width: auto;
+  height: auto;
+  max-height: 600px;
+  background-color: #c8b3e2;
+  overflow: auto;
```
Comment (usatovaDasha) [inline test.css:36]: no need this line

File: test.js [hunk 7]
```diff
@@ -0,0 +1,154 @@
+oldTasks = JSON.parse(localStorage.getItem("todoList"));
```
Comment (usatovaDasha) [inline test.js:1]: const

File: test.js [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+let allTasks = oldTasks || [];
+let valueInput = "";
+let input = null;
+let editTask = null;
+
+window.onload = function init() {
```
Comment (usatovaDasha) [inline test.js:7]: es6 function

File: test.js [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+  input = document.getElementById("textId");
+  input.addEventListener("change", updateValue);
+  render();
+};
+
+pushButton = () => {
```
Comment (usatovaDasha) [inline test.js:13]: const pushButton

File: test.js [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+  allTasks.push({
+    text: valueInput,
+    isCheck: false,
+  });
+  render();
+  valueInput = "";
+  input.value = "";
+};
+
+updateValue = (event) => {
```
Comment (usatovaDasha) [inline test.js:23]: add const before names functions

File: test.js [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+  valueInput = event.target.value;
+};
+
+render = () => {
+  let content = document.getElementById("content-page");
```
Comment (usatovaDasha) [inline test.js:28]: const

File: test.js [hunk 12]
```diff
@@ -0,0 +1,154 @@
...
+  while (content.firstChild) {
+    content.removeChild(content.firstChild);
+  }
+  allTasks.sort((a, b) =>
```
Comment (usatovaDasha) [inline test.js:32]: => a.isCheck - b.isCheck

File: test.js [hunk 13]
```diff
@@ -0,0 +1,154 @@
...
+    a.isCheck > b.isCheck ? 1 : a.isCheck < b.isCheck ? -1 : 0
+  );
+
+  allTasks.forEach((item, index) => {
+    let container = document.createElement("div");
```
Comment (usatovaDasha) [inline test.js:37]: const

File: test.js [hunk 14]
```diff
@@ -0,0 +1,154 @@
...
+    container.id = `task-${index}`;
+    container.className = "task-container";
+
+    const checkbox = document.createElement("input");
+    checkbox.type = "checkbox";
+    checkbox.checked = item.isCheck;
+    checkbox.className = "checkboxParams";
+    checkbox.onchange = function () {
```
Comment (usatovaDasha) [inline test.js:45]: es6

File: test.js [hunk 15]
```diff
@@ -0,0 +1,154 @@
...
+      onChangeCheckBox(index);
+    };
+    let checkboxDiv = document.createElement("div");
```
Comment (usatovaDasha) [inline test.js:48]: const

File: test.js [hunk 16]
```diff
@@ -0,0 +1,154 @@
...
+    checkboxDiv.className = "checkboxDivClass";
+    checkboxDiv.appendChild(checkbox);
+    container.appendChild(checkboxDiv);
+
+    let imageEdit = document.createElement("img");
```
Comment (usatovaDasha) [inline test.js:53]: const

File: test.js [hunk 17]
```diff
@@ -0,0 +1,154 @@
...
+    imageEdit.src = "edit.png";
+    imageEdit.className = "picParamsEdit";
+    imageEdit.id = `task-${index}-edit`;
+    imageEdit.onclick = function () {
```
Comment (usatovaDasha) [inline test.js:57]: es6

File: test.js [hunk 18]
```diff
@@ -0,0 +1,154 @@
...
+      editTask = index;
+      render();
+    };
+    if (allTasks[index].isCheck === true) {
```
Comment (usatovaDasha) [inline test.js:61]: if (allTasks[index].isCheck)

File: test.js [hunk 19]
```diff
@@ -0,0 +1,154 @@
...
+
+      let newInput = document.createElement("input");
+      newInput.type = "text";
+      newInput.value = item.text;
+      newInput.id = "inputEdit";
+      newInput.addEventListener("change", startEditTask);
+      // newInput.addEventListener("blur", stopEditTask);
+      container.appendChild(newInput);
+
+      let newButton1 = document.createElement("button");
```
Comment (usatovaDasha) [inline test.js:91]: const variables

File: test.js [hunk 20]
```diff
@@ -0,0 +1,154 @@
...
+    editDiv.appendChild(imageEdit);
+    container.appendChild(editDiv);
+
+    let deleteDiv = document.createElement("div");
+    deleteDiv.className = "deleteDivClass";
+    deleteDiv.appendChild(imageDelete);
+    container.appendChild(deleteDiv);
+
+    content.appendChild(container);
+    console.clear;
```
Comment (usatovaDasha) [inline test.js:125]: remove console

File: test.js [hunk 21]
```diff
@@ -0,0 +1,154 @@
...
+};
+
+onChangeCheckBox = (index) => {
+  allTasks[index].isCheck = !allTasks[index].isCheck;
+  render();
+};
+
+startEditTask = (event) => {
+  saveTasks = allTasks[editTask].text;
+  allTasks[editTask].text = event.target.value;
```
Comment (usatovaDasha) [inline test.js:136]: do you change it right away?

**Comment by tolyanchenko** [comment]
some fixes were added

File: test.js [hunk 22]
```diff
@@ -34,9 +34,10 @@ render = () => {
   );
 
   allTasks.forEach((item, index) => {
-    let container = document.createElement("div");
-    container.id = `task-${index}`;
+    const container = document.createElement("div");
+    container.id = "task - " + index;
```
Comment (usatovaDasha) [inline test.js:38]: es6 string

File: test.js [hunk 23]
```diff
@@ -34,9 +34,10 @@ render = () => {
...
     container.className = "task-container";
+    console.log(container.id);
```
Comment (usatovaDasha) [inline test.js:40]: remove

File: test.js (hunk 14 above)
Comment (usatovaDasha) [reply test.js:46]: !!!!!

File: test.js [hunk 24]
```diff
@@ -45,24 +46,24 @@ render = () => {
...
     checkboxDiv.className = "checkboxDivClass";
     checkboxDiv.appendChild(checkbox);
     container.appendChild(checkboxDiv);
 
-    let imageEdit = document.createElement("img");
+    const imageEdit = document.createElement("img");
     imageEdit.src = "edit.png";
     imageEdit.className = "picParamsEdit";
-    imageEdit.id = `task-${index}-edit`;
+    imageEdit.id = "task - " + index;
```
Comment (usatovaDasha) [inline test.js:57]: es6

**Comment by tolyanchenko** [comment]
some more fixes were added

**Pull request merged**